_data/apis.yml

```diff
@@ -18,7 +18,7 @@
     - xml
 - title: WONDER
   id: wonder
-  url: http://wonder.cdc.gov?request_xml=
+  url: htps://wonder.cdc.gov/ controller/datarequest/[database ID]
```

Comment:
This looks to have a typo, shouldn't this be `https` instead of `htps`? Also, there seems to be a space between `/` and `controller`.

Reply:
Thanks Drew and @wap2git , I'll update to "https://wonder.cdc.gov/controller/datarequest/[database ID]" in the merge.